src/test/java/org/folio/edgecommonspring/security/SecurityManagerServiceTest.java

```diff
@@ -25,53 +30,139 @@ class SecurityManagerServiceTest {
 
   private static final String MOCK_TOKEN = "eyJhbGciOiJIUzI1NiJ9eyJzdWIiOiJ0ZXN0X2FkbWluIiwidXNlcl9pZCI6ImQyNjUwOGJlLTJmMGItNTUyMC1iZTNkLWQwYjRkOWNkNmY2ZSIsImlhdCI6MTYxNjQ4NDc5NCwidGVuYW50IjoidGVzdCJ9VRYeA0s1O14hAXoTG34EAl80";
```

Comment:
I realize this is not part of this change, but I really dislike using what appear to be secrets in the source code. I know these aren't real secrets but they look like them and I think that sends the wrong message. I don't think we need an actual token here. In other words, parsing the token isn't part of the test unless I am mistaken.

Reply:
changed it to MOCK_TOKEN = "test-token"